docs/kratos/passwordless/06_code.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-id: one-time-code
+id: code
```

Comment:
Revert or vercel.json

also i think "email code" or "email-sms-code" would be better. or "magic code"